README.md

```diff
@@ -3,6 +3,7 @@
 </a>
 
 [![Build Status](https://github.com/eth-cscs/reframe/workflows/ReFrame%20CI/badge.svg)](https://github.com/eth-cscs/reframe/actions?query=workflow%3A%22ReFrame+CI%22)
+[![GitHub Super-Linter](https://github.com/eth-cscs/reframe/workflows/Lint%20Code%20Base/badge.svg)](https://github.com/marketplace/actions/super-linter)
```

Comment:
Can you copy paste here how this looks like?

Reply:
It looks like this: https://github.com/github/super-linter#add-super-linter-badge-in-your-repository-readme